sipsorcery/sipsorcery
# UnitTests: Logging

to harmonise with existing test-logging:
- added ctor(ITestOutputHelper) everywhere
- add ILogger logger = SIPSorcery.Sys.Log.Logger everywhere
- change Console.WriteLine to logger.LogDebug
- added logging for entering method to every test

Fix MockSIPDNSManager <- should fix test-fails from PR#135
added "Skip" to DNS.QType.Any test <- should fix test-fails from PR#135
